docs/classic-ui/tinymce-customization.md

```diff
@@ -0,0 +1,87 @@
+---
+myst:
+  html_meta:
+    "description": "TinyMCE customization in Plone 6"
+    "property=og:description": "TinyMCE customization in Plone 6"
+    "property=og:title": "TinyMCE customization in Plone 6"
+    "keywords": "Plone 6, Classic UI, Bootstrap 5, TinyMCE, customization"
+---
+
+(classic-ui-tinymce-customization-label)=
+
+# TinyMCE customization
+
+This chapter is a developer reference manual for customizing {term}`TinyMCE`.
+
+
+## Customize {menuselection}`Format` menu
+
+There are several options to customize TinyMCE's {menuselection}`Format --> Formats` menu.
+
+The first two options use TinyMCE's [`formats` JSON configuration](https://www.tiny.cloud/docs/tinymce/latest/content-formatting/).
+They are less complicated to implement than the third option.
+
+
+### Add-on GenericSetup configuration file
+
+This option is best for system administrators and developers who write their own add-ons to ease reproducibility.
+
+You can add a GenericSetup configuration file to your add-on, such as {file}`profiles/default/registry/tinymce.xml`, with the following content.
+
+```xml
```

Comment:
What about
```xml
  <records interface="Products.CMFPlone.interfaces.ITinyMCESchema" prefix="plone">
    <value key="other_settings">
      {
        "extended_valid_elements":"i[class],script[type|src]",
        "style_formats": [
          {
            "title": "Highlight",
            "block": "p",
            "attributes": {
              "class": "featured"
            }
          },
          {
            "title": "Alert",
            "block": "p",
            "attributes": {
              "class": "redalert"
            }
          }
        ],
        "style_formats_merge": true
      }
    </value>
  </records>
```

Reply:
Beware: `other_settings` will override all previously made settings. This is meant to inject settings which are not populated in other config fields.

And the `style_formats_merge` options provides the default "tinymce styles" but that's not exactly the same as the plone default tinymce styles ...